src/test/java/core/FormValidationTest.java

```diff
@@ -75,8 +74,7 @@ private void jsValidation() {
     }
 
     private void navigateAway() {
-        jenkins.runThenConfirmAlert(
-                () -> driver.findElement(By.id("jenkins-head-icon")).click());
+        jenkins.runThenHandleUserPrompt(() -> jenkins.open());
```

Comment:
This sleep below remains suspicious, but that code has been here for many, many years.